tasks/eth/4-protocol-versions/README.md

```diff
@@ -0,0 +1,204 @@
+# Mainnet Protocol Versions Update - 5.0.0 (Delta)
+
+## Objective
+
+This is the playbook for updating the protocol versions of the `ProtocolVersions` contract on Ethereum mainnet to 5.0.0 (Delta final).
+
+Both versions are currently set to 3.1.0 (Regolith), which are the versions set at deployment.
+
+Both are set at the same time in a batch because the time window has passed in which it makes sense to set the recommended
+version earlier. This isn't an issue, as both are informative signals.
+
+## TX Creation
+
+Transactions were created using
+```
+export PV_ENC=$(cd ../../../lib/optimism && go run ./op-chain-ops/cmd/protocol-version encode --major 5)
+# 0x0000000000000000000000000000000000000005000000000000000000000000 
+export PV_ADDR=0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935 
+just add-transaction $PV_ADDR 'setRecommended(uint256)' $PV_ENC
+just add-transaction $PV_ADDR 'setRequired(uint256)' $PV_ENC
+```
+This batches setting both, the recommended and required versions in one multicall.
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
+just install
+cd tasks/eth/4-protocol-version
```

Comment:
```suggestion
cd tasks/eth/3-protocol-version
```

Reply:
Moved it to `03-protocol-versions-delta`. In anticipation of there being more than 10 tasks, it probably makes sense to move all tasks into two-digit prefixed folders, which gives a better lexicographical ordering.